packages/apollo-engine-reporting/src/plugin.ts

```diff
@@ -124,7 +146,9 @@ export const plugin = <TContext>(
           document: requestContext.document,
           source: requestContext.source,
           trace: treeBuilder.trace,
-          schemaHash,
+          executableSchemaId: executableSchemaIdGenerator(
```

Comment:
There is not currently a mechanism for this plugin to every report more than one schema over the course of the server's existence, right? So why not just calculate this once (in serverWillStart)?

It seems like what you've implemented allows the schema to change over time for the purpose of trace reports, but not for the purpose of schema reporting.  Is this intentional? (eg, is fixing this part of the "supporting federation" project?) At the very least this could be called out with a comment, noting that we are sensitive to changes in schema here but not in the reporting feature (and that retaining that sensitivity is a goal which is why we don't just calculate this once in serverWillStart).